src/app/page.tsx

```diff
@@ -1,10 +1,17 @@
 import { Metadata } from 'next'
+import dynamic from 'next/dynamic'
 import { FC, Suspense } from 'react'
 
 import BooksFilters from '@/app/catalog/components/book-filters'
-import BooksList from '@/app/catalog/components/books-list'
 import { Filters } from '@/types/book'
 
+const DynamicBooksList = dynamic(
+  () => import('@/app/catalog/components/books-list'),
+  {
+    loading: () => <p>Завантаження...</p>,
```

Comment:
```suggestion
   loading: () => (
      <div className='col-span-3 flex flex-col items-center justify-center h-[600px] text-center  '>
        <Loader2 className='col-span-3 self-center w-12 h-12 animate-spin text-gray-300' />
        <span className='text-2xl text-gray-300 font-bold mb-2'>
          Завантаження...
        </span>
      </div>
    ),
```
Here it need make design

Reply:
Fixed